browser/components/newtab/aboutNewTabService.js

```diff
@@ -37,6 +36,17 @@ const PREF_MATCH_OS_LOCALE = "intl.locale.matchOS";
 // The preference that tells what locale the user selected
 const PREF_SELECTED_LOCALE = "general.useragent.locale";
 
+// The preference that tells what remote mode is enabled.
+const PREF_REMOTE_MODE = "browser.newtabpage.remote.mode";
+const MODE_CHANNEL_MAP = {
+  "production": {origin: "https://newtab.cdn.mozilla.net"},
```

Comment:
production origin should be https://content.cdn.mozilla.net and stage should be: https://content-cdn.stage.mozaws.net

https://bugzilla.mozilla.org/show_bug.cgi?id=1247342